src/main/java/pro/belbix/tgnotifier/tg/Commands.java

```diff
@@ -68,33 +68,71 @@
         + "Set value to 0 for unsubscribe.";
     public final static String UNKNOWN_COMMAND = "Incorrect or unknown command. Use " + HELP;
 
-    public static String responseForCommand(String command) {
+    public static UserResponse responseForCommand(String command) {
         if (command == null) {
-            return UNKNOWN_COMMAND;
+            return new UserResponse(UNKNOWN_COMMAND, null);
         }
         switch (command) {
             case FARM_CHANGE:
-                return FARM_CHANGE_DESC + "\n" + PERCENT_VALUE_CALLBACK;
+                InlineButton[] buttons_farmChange = {
```

Comment:
please, use the java name convention for all variables, like buttonsFarmChange 
https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html#:~:text=Class%20names%20should%20be%20nouns,such%20as%20URL%20or%20HTML).

Reply:
Changed